app/pages/cif/projects.tsx

```diff
@@ -53,21 +54,13 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
     const response = await commitProjectMutation(preloadedQuery.environment, {
       input: {},
     });
-    await router.push({
-      pathname: "/cif/create-project",
-      query: {
-        id: response.createProject.projectRevision.id,
-      },
-    });
+    await router.push(
+      getProjectRevisionPageRoute(response.createProject.projectRevision.id)
```

Comment:
do we need to handle the case where `createProject` fails?

Reply:
Probably, but I'd like to switch to using the `useMutation` hook everywhere first